### PR TITLE
Migrate API permissions - 17.11

### PR DESCRIPTION
**Pull request by jamesm-macrometa**
This PR is to migrate the API permissions article from James B's KB.

JIRA: https://macrometa.atlassian.net/browse/DOC-300

### CODE REVIEW

**Comment by jamesm-macrometa** [comment]
I think this section could still use more detail about different access levels and how they interact with each other, but this is a good start.

File: docs/account-management/permissions/edit-permissions.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+title: Edit Permissions
+---
+
+You can change permissions for users and API keys from the GDN console.
+
+## Change Permissions for Users or API Keys
+
+Once you log in to a Macrometa account, you can view all user accounts and API keys that you have access to.
+
+1. Log in to Macrometa.
```
Comment (diana-macrometa) [inline docs/account-management/permissions/edit-permissions.md]: Add a link to the Macrometa login screen. You can grab that from another topic, lots in Collections and Query Workers.

File: docs/account-management/permissions/edit-permissions.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+1. Click **Account**.
+1. If you want to edit permissions for a user, click the **Users** tab and click the user in the list. Alternatively, click the **API Keys** tab and click the API key in the list.
```
Comment (diana-macrometa) [inline docs/account-management/permissions/edit-permissions.md]:
```suggestion
1. If you want to edit permissions for a user, then click the **Users** tab and click the user in the list. Alternatively, click the **API Keys** tab and click the API key in the list.
```

File: docs/account-management/permissions/edit-permissions.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+sidebar_position: 20
+title: Edit Permissions
+---
+
+You can change permissions for users and API keys from the GDN console.
+
+## Change Permissions for Users or API Keys
```
Comment (diana-macrometa) [inline docs/account-management/permissions/edit-permissions.md]: Please rewrite this as two separate tasks.

Change User Permissions

Change API Key Permissions

File: docs/account-management/permissions/edit-permissions.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+Once you log in to a Macrometa account, you can view all user accounts and API keys that you have access to.
+
+1. Log in to Macrometa.
+1. Click **Account**.
+1. If you want to edit permissions for a user, click the **Users** tab and click the user in the list. Alternatively, click the **API Keys** tab and click the API key in the list.
+1. On the Permissions tab, select the radio button for the permissions you want to assign.
```
Comment (diana-macrometa) [inline docs/account-management/permissions/edit-permissions.md]: Link to the topic where permissions are defined.

File: docs/account-management/permissions/edit-permissions.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+1. Save the details for the user.
```
Comment (diana-macrometa) [inline docs/account-management/permissions/edit-permissions.md]: What do I click?

Comment (diana-macrometa) [reply docs/account-management/permissions/edit-permissions.md]: Also, this only applies to users.

**Review by diana-macrometa — CHANGES_REQUESTED** [review]
This will be a great addition to the docs and really help our users! I have a few suggestions.

**Comment by jamesm-macrometa** [comment]
Thanks @diana-macrometa !  Incorporated your feedback.  Let me know if you spot anything else!  Potential pending change on the "stacked dots" name.

File: docs/account-management/permissions/change-user-permissions.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+## Change Permissions for Users
+
+Once you log in to a Macrometa account, you can view all user accounts and API keys that you have access to.
+
+1. Log in to your [Macrometa account](https://auth.paas.macrometa.io/).
+1. Click **Account**.
+1. Click the **Users** tab and find the user for which you want to change permissions.
+1. Click the stacked dots on the right side to open the drop-down menu.
+1. Click **Edit User**.
+1. On the Permissions tab, select the radio button for the permissions you want to assign.  Refer to [Permissions](index.md) for information about each permission level.
```
Comment (diana-macrometa) [inline docs/account-management/permissions/change-user-permissions.md]:
```suggestion
1. On the Permissions tab, select the radio button for the permissions you want to assign. Refer to [Permissions](index.md) for information about each permission level.
```

**Review by diana-macrometa — APPROVED** [review]
Looks good. One super minor edit.

Please create JIRA tickets to add SDK, CLI, and API tabs to these tasks. 

A lot of the SDK content already exists, like [Set Permissions](https://macrometa.com/docs/sdks/set-permissions) and [Create API Key}(https://macrometa.com/docs/sdks/create-api-key), we just need to organize it.

**Comment by jamesm-macrometa** [comment]
Hey @diana-macrometa - for some reason the changes I made to this branch don't appear in the [release-17-11 PR](https://github.com/Macrometacorp/docs/pull/281) despite this branch having been merged into that one.  Any idea why this might be the case?

**Comment by diana-macrometa** [comment]
I might have accidentally removed them in a merge conflict. Most of those topics do not exist anymore since I did my rework to combine the stuff that was in the SDKs. I tried to make sure I captured all content changes, it's possible I missed something. Sorry!

**Pull request merged**